src/Core/Containers/Grid.inl

```diff
@@ -127,7 +127,7 @@ T& Grid<T, D>::at( const typename Grid<T, D>::Iterator& it ) {
     CORE_ASSERT( it.getGridSize() == m_size, "Incompatible iterator" );
     return at( it.getLinear() );
 }
-
+*/

```

Comment:
If this method is no longer used, remove it instead of just commenting.

Reply:
restored